extensions/administration.py

```
@@ -574,12 +573,15 @@ async def database_sync(self, ctx: commands.Context, url: str | None = None) ->
             with open("temp_import.sql", "wb") as f:
                 f.write(content)
         except Exception as e:
-            await status_msg.edit(
-                content=tanjunLocalizer.localize(self._locale(ctx), "commands.admin.database_sync.download_error", error=e)
-            )
-            return
+                await status_msg.edit(
```

Flake8 (7.3.0)

[error] over-indented

(E117)